### ISSUE TITLE
Package maintenance

### ISSUE THREAD

**Issue by rudiedirkx** [LaravelCollective/annotations#124]
I've asked everywhere and nobody is responding, so this will be the last time:

I'd like to help/take over maintaining this annotations package. I'd hate to see it disappear, because I love it.

----

For now, I've created a fork with all PRs included: https://github.com/rudiedirkx/laravelcollective-annotations but it would be much better if this actual package survived.

You can use the fork by adding a custom repository in your project's `composer.json`:

    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/rudiedirkx/laravelcollective-annotations.git"
        }
    ],

and then require that version specifically:

    "laravelcollective/annotations": "dev-8.0-fork as 8.1",

It works, but it's silly, but I hope it won't be necessary for long.

**Comment by Putr** [comment]
Damn ... I really wish they would let you take it over. 

If nothing else ... we need Laravel9&10 support.

Can I help you add Laravel 9 support on your branch, while we try and get you access to the repo?

**Comment by rudiedirkx** [comment]
It has Laravel 9 support in branch `8.0-fork`. I'm not on L 10 yet, so I can't test that, but I assume it would work for L 10 too.

I can't reach anyone with maintainer access, so I don't see any development ahead =( You're very welcome to try of course.

**Comment by Putr** [comment]
Thank you for the 8.0-fork info, you just saved me A LOT of headache. 

I've sent an email – as have you, probably. I'll see what else I can do. Not only that, but I depend on this project a lot, so when you take over (yes, I'm very optimistic) I'm willing to help out.

**Comment by loonytoons** [comment]
@mlantz anything you can do to help here?

**Comment by rudiedirkx** [comment]
I've updated `8.0-fork` to include laravel 10 support, because that seems to work. I've tested only briefly, and only with Doctrine annotations, not PHP Attributes.

**Comment by letharion** [comment]
I just found https://laravelcollective.com/thanks, which in wayback machine appears for the first time in May.
It seems like the whole namespace is just abandoned, so I guess it's time to move on.
The fork has served me well for some time, but I guess I will take a look at the recommended alternative spatie/laravel-route-attributes and see if that's a reasonable option.

**Comment by rudiedirkx** [comment]
They also abandoned `laravelcollective/html` and recommend `spatie/laravel-html`, but those aren't compatible at all. Such a waste. I'll never understand why they didn't just accept our help and keep laravelcollective alive =(

**Comment by letharion** [comment]
It's a pretty sad way to go, when you specifically are asking to help.
@rudiedirkx Do you intend to maintain the fork going forward? It would be nice if this could continue, albeit under a new package name. Since I currently use it extensively I could at the very least provide help testing it.

**Comment by rudiedirkx** [comment]
Yes, I want to keep using it, so I'll maintain it, but there's not much maintenance necessary I think.

**Comment by rudiedirkx** [comment]
I've changed how requiring my fork works. Instead of `"repositories"` and `"dev-8.0-fork as 8.1"` like I said in the OP, now you:

1. `composer require rdx/laravelcollective-annotations`
2. remove the `"repositories"` entry for this repo
3. `composer remove laravelcollective/annotations`

That installs stable [`rdx/laravelcollective-annotations`](https://packagist.org/packages/rdx/laravelcollective-annotations) `8.2` from Packagist, and replaces `laravelcollective/annotations`. All package code still works. [`"replace"`](https://github.com/rudiedirkx/laravelcollective-annotations/blob/8.0-fork/composer.json#L9) is a Composer feature exactly for this purpose.

You only have to do this when you `upgrade/update/require/remove` something in your `composer.json`. If your composer.json doesn't change, you're fine, but next time it does, don't forget this. The `"repositories"` trick won't work anymore.

----

I've done the same for `laravelcollective/html`: require `rdx/laravelcollective-html` (`6.6`) and remove `laravelcollective/html`. All other packages will think `laravelcollective/html` is installed, and everything works like magic. Composer (and `"replace"`) is brilliant.
